src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -327,6 +331,28 @@ describe('CdkOption', () => {
       expect(fixture.componentInstance.changedOption).toBeDefined();
       expect(fixture.componentInstance.changedOption.id).toBe(optionInstances[0].id);
     });
+
+    it('should focus and toggle the next item when pressing SHIFT + DOWN_ARROW', () => {
+      let selectedOptions = optionInstances.filter(option => option.selected);
+      const downKeyEvent = createKeyboardEvent('keydown', DOWN_ARROW);
+      Object.defineProperty(downKeyEvent, 'shiftKey', {get: () => true});
```

Comment:
You should pass the modifiers directly to `createKeyboardEvent` (so we avoid the `defineProperty` call here).